README-cn.md

```diff
@@ -142,20 +160,14 @@ python examples/<example_name>.py
 
 更详细的示例和教程请在 `notebooks` 和 [notebooks/README.md](notebooks/README.md)中寻找。 
 
-## 您的贡献
-添加新功能，改进文档，修复错误或编写教程都是有用的贡献的例子。此外，如果您要发布新的攻击或防御，我们建议您将其添加到Adversarial Robustness Toolbox中，以便其他人可以评估。
-您可以通过GitHub拉取启动错误修复。在为Adversarial Robustness Toolbox提供代码贡献时，我们希望您遵循`PEP 8`编码标准，并为新功能提供单元测试。
-该项目使用[DCO](https://developercertificate.org/)。请务必使用`-s`标志注销提交，或在提交消息中添加`Signed-off-By：Name <Email>`。
+##為ART貢獻
 
-#### 示例
-```bash
-git commit -s -m 'Add new feature'
-```
+看到 [CONTRIBUTING-cn.md](CONTRIBUTING-cn.md)
 
 ## 引用ART
 如果您使用ART进行研究，请考虑引用以下参考文件：
 ```
-@article{art2018,
+@article{art2018,~~~~
```

Comment:
Do we need these 4 tilde characters?

Reply:
no I don't think we do. I didn't realise that was inserted. I just removed them